realm/realm-library/src/main/java/io/realm/RealmFieldType.java

```diff
@@ -42,17 +42,16 @@
     FLOAT(9),
     DOUBLE(10),
     OBJECT(12),
-    LIST(13);
-    // BACKLINK(14); Not exposed until needed.
+    LIST(13),
+    BACKLINK(14);
```

Comment:
I just uncommented it, but I think that we should probably be consistent.  Btw, I think that this is probably a breaking change...